services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/AutowiredDeviceConnectionAdapter.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.deviceregistry;
+
+import org.eclipse.hono.service.deviceconnection.DeviceConnectionService;
+import org.eclipse.hono.service.deviceconnection.EventBusDeviceConnectionAdapter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+/**
+ * A default event bus based service implementation of the {@link DeviceConnectionService}.
+ * <p>
+ * This wires up the actual service instance with the mapping to the event bus implementation. It is intended to be used
+ * in a Spring Boot environment.
+ */
+@Component
+public final class AutowiredDeviceConnectionAdapter extends EventBusDeviceConnectionAdapter<Void> {
+
+    private DeviceConnectionService service;
+
+    @Autowired
+    @Qualifier("serviceImpl")
```

Comment:
Note: The other `Autowired*Adapter` classes have the `backend` qualifier here, but since there is no need for a unifying `DeviceConnectionBackend`, there is no separate `backend` bean used here.